blockchain/fundamentals/ethereum-ii/erc20-tokens.md

```diff
@@ -11,15 +11,86 @@ type: normal
 
 category: must-know
 
+standards:
+  blockchain.compare-altcoins.1: 10
+  blockchain.compare-altcoins.5: 20
+
 aspects:
 
   - introduction
 
 ---
-# ERC20-Tokens
+# ERC20 Tokens
 
 ---
 ## Content
 
+**ERC20** is the standard used to represent currencies. It allows for basic account functionalities like transfer, get address balance, or get total supply.
+
+The basic ERC20 interface looks like this[1]:
+```
+contract ERC20BasicInterface {
+  function totalSupply() public 
```

Comment:
in the solidity workout I think you never started explaining what these keywords mean : public, view, uint etc

Reply:
Yeah so the thing is I kinda imagined a second blockchain course on Solidity in which the language perks would be explained in detail (much like keywords in java).

Reply:
This course should be less of the specifics of any platform/programming language and more about the generalities of the blockchain technology